src/mc_tools.jl

```diff
@@ -10,10 +10,7 @@ References
 
 http://quant-econ.net/jl/finite_markov.html
 =#
-
-# new method to check if all elements of an array x are equal to p
-isapprox(x::Array, p::Number) = all([isapprox(x[i], p) for i=1:length(x)])
-isapprox(p::Number, x::Array) = isapprox(x, p)
+import LightGraphs: DiGraph, period, attracting_components, strongly_connected_components
```

Comment:
We need to make a decision regarding imports.

For the rest of the code we have put all `using` statements in the main module file (`src/QuantEcon.jl`). 

Here it is a little different because you are adding a new method to the `period` function. 

I think I would prefer that we do `using LightGraphs: DiGraph, attracting_components, strongly_connected_components` in (`src/QuantEcon.jl`) and then define the method below 

```julia
function LightGraphs.period(mc::MarkovChain)
    ...
```

I'm not 100% convinced of this, what thoughts do others have?

If we do make the proposed decision, I will probably not `import` anything from `Base` and add new methods using the syntax `Base.function_name` -- over time I find that I am preferring this explicit form of adding new methods